SConstruct

```diff
@@ -40,17 +40,18 @@ import os
 import re
 import sys
 import glob
+import locale
 import platform
 import shutil
 import subprocess
 import distutils.dir_util
 
-if os.name == "nt":
-	import _locale
-	_locale._getdefaultlocale_backup = _locale._getdefaultlocale
-	_locale._getdefaultlocale = (lambda *args: (_locale._getdefaultlocale_backup()[0], 'utf8'))
-
-EnsureSConsVersion( 3, 0, 2 )  # Substfile is a default builder as of 3.0.2
```

Comment:
It probably doesn't do any harm these days to remove this, but checking that it was intentional to remove the minimum version test.

Reply:
Thanks for spotting that - it was unintentional. Restored in https://github.com/GafferHQ/gaffer/pull/5194/commits/54f0863f8d96e3afa933e4651a4626e9a0b8ec9f.